storage/storage.go

```diff
@@ -464,27 +466,25 @@ func (storage DBStorage) getReportUpsertQuery() string {
 	case types.DBDriverSQLite3:
 		return `INSERT OR REPLACE INTO report(org_id, cluster, report, reported_at, last_checked_at, kafka_offset)
 		 VALUES ($1, $2, $3, $4, $5, $6)`
-	case types.DBDriverPostgres:
-		return `INSERT INTO rule(org_id, cluster, report, reported_at, last_checked_at, kafka_offset)
+	default:
```

Comment:
If doing this refactor will be much more better use regular if, like:
```
if storage.dbDriverType == types.DBDriverSQLite3 {
...
}
return `....`
```

Reply:
done